### ISSUE TITLE
High training loss, infinite evaluation loss and evaluation wer always equal to 1

### ISSUE THREAD

**Issue by t-khanusa** [khanld/ASR-Wa2vec-Finetune#12]
Hi Mr Khanh, I tried to fine-tune the wav2vec2 model with the code you provided. I set up the same dataset structure as yours, just fix line number 18 dataset;

labels_batch = self.processor.tokenizer(transcripts, padding="longest", return_tensors="pt")

Then proceed to fine-tune, my data has about 2 hours of Vietnamese audio. However, after training with quite a few epochs, the loss is still very high and the wer on the val set does not change (1.00). What do I need to check and fine-tune to achieve good results in this task? Thank you very much

<img width="1444" height="384" alt="Image" src="https://github.com/user-attachments/assets/4a9ae012-a0bb-49c9-8001-3f19f288d2d8" />

<img width="1015" height="661" alt="Image" src="https://github.com/user-attachments/assets/2fcb02c5-7aba-4a7e-9370-e7444a9031fe" />
<img width="953" height="345" alt="Image" src="https://github.com/user-attachments/assets/962e9c68-1b91-41be-8a91-4b09766e7824" />

<img width="1228" height="178" alt="Image" src="https://github.com/user-attachments/assets/52639144-a8f9-4523-a0e0-36199f4edf7a" />
